### PR TITLE
Add "unspecified" (指定なし) to FireZone #18

### PR DESCRIPTION
**Pull request by hirao-k**
#18

### CODE REVIEW

File: src/app/core/building/building.component.html [hunk 1]
```diff
@@ -89,7 +89,7 @@ <h3 class='red'>※ 入力必須項目</h3>
               <mat-form-field class="full-width-input" appearance='outline'>
                 <mat-label>防火地域</mat-label>
                 <mat-select [(ngModel)]='blueprint.fireZone' id="fireZone" name="fireZone" #fireZone="ngModel" required>
-                  <mat-option value="null">防火地域</mat-option>
```
Comment (thanhtetaung) [inline src/app/core/building/building.component.html:92]: @hirao-k Is blueprint analysis execution API already supported **unspecified** for fire zone?

Comment (hirao-k) [reply src/app/core/building/building.component.html:92]: @thanhtetaung   I haven't.  After I deploy, I'll tell you that.

Comment (hirao-k) [reply src/app/core/building/building.component.html:92]: @thanhtetaung  I deployed it to API.

Comment (thanhtetaung) [reply src/app/core/building/building.component.html:92]: @hirao-k thanks.

**Review by thanhtetaung — APPROVED** [review]

**Pull request merged**